BalanceSheet.py

```diff
@@ -548,170 +420,154 @@ def quickRatio( cashEquivalents, aRecievable, currentLiabilities):
 ##Current Liabilities:
 
 currentLiabilities = [ "AccountsPayable", "other current Liabilities", "Deferred revenue", "Commercial paper", "Term Debt"]
-cL2020= [42296, 42684, 6643, 4996, 8773]
+cl2020= [42296, 42684, 6643, 4996, 8773]
 
 # Noncurrent Liabilities
 nonCurrentLiabilities = ["Term debt", "Other non-current Liabilities"]
-nCL2020 = [98667, 54490]
+ncl2020 = [98667, 54490]
 
-subTotalCurrentLiabilities2020 = iterateSubtotals(cL2020)
+subTotalCurrentLiabilities2020 = iterateSubtotals(cl2020)
 
-subTotalNonCurrentLiabilities2020 = iterateSubtotals(nCL2020)
+subTotalNonCurrentLiabilities2020 = iterateSubtotals(ncl2020)
 
 totalLiabilities = subTotalCurrentLiabilities2020+ subTotalNonCurrentLiabilities2020
 
 print("totalLiabilities = ", totalLiabilities )
 
- 
 
 #---
-#Capital
-## Commitments & Contingencies
+#Capital : is about
+## Commitments & Contingencies , a.k.a
 ## Shareholder's equity
 
 shareholderEquity = ["Common Stock", "Retained Earnings", "Accumulated Other comprehensive income (loss)"]
-#
+
 shEquity2020 = [50779,14966, -406]
 
 shEquity2019 = [45174, 45898, -584]
 
-subTotalEquity2020 = iterateSubtotals(shEquity2020) # get equity 
-print("subTotalEquity2020 = ", subTotalEquity2020)
-totalEquity = subTotalEquity2020
+subTotalEquities2020 = iterateSubtotals(shEquity2020) # get equity 
+print("subTotalEquity2020 = ", subTotalEquities2020)
+totalEquities = subTotalEquities2020
 # subEquity2019 = iterateSubtotals(shEquity2019)
 
-
-#def __init__(self):
-#        pass
-
-#def __repr__(self):
-    #pass
-    
-
+#---
 #def calc_grossMargin():
-
-# Income Statement [#TODO: move to new  module]
+#class Income Statement 
         
 def calcCogs(subTotals):
 
     cogs = iterateSubtotals(subTotals)
     return cogs
-
+#def caclGrossMargin():
+#TODO: move to new  module]
+#---
+    
 # Balnace sheet [static ] methods 
 
-def isEqual(totalAssets, totalLiabilities, totalEquity ):
+def isEqual(totalAssets, totalLiabilities, totalEquity ): # +
     """
- # applying basic accounting equation:
-        Assets  = Liabilities + Equity
+     Applies the `Accounting Equation`
+    Assets  = Liabilities + Equity
+    """ 
+    condition = totalAssets + totalLiabilities == totalEquity
+    return condition 
+
+def getDifference(LHS2, RHS2): # - 
+    """for networth 
+        totalAssets + totalLiabilities == totalEquities """
+    return abs(LHS2) - abs(RHS2)
+
+def setRhsLhs(totalAssets, totalLiabilities, totalEquities ): #+ 

```

Comment:
adds `setRhsLhs` to ease up on any further functions that require `Accounting Equation`
Rhs = `Assets` while LHS = `Liabilities` & `Equities`